docs/admin/setup/legal/privacy-statement-templates.rst

```diff
@@ -1,6 +1,7 @@
 Privacy Statement German template
-"""""""""""""""""""""""""""""""""
-.. code-block:: none
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: markdown
```

Comment:
Before these Changes, the templates were not highlighted at all. Now, the German privacy statement (top of image) is highlighted differently compared to the English version (bottom of image) and the imprint templates (headings not highlighted):
![Screenshot 2023-10-16 at 16 17 24](https://github.com/ls1intum/Artemis/assets/47261058/2a275b4d-4ecc-4ca6-b618-e64e4ee576a3)